tsconfig.json

```diff
@@ -11,7 +11,16 @@
     "strict": true,
     "noImplicitAny": false,
     "target": "esnext",
-    "types": ["react", "react-native", "react-redux", "jest"],
+    "types": [
+      "react",
+      "react-native",
+      "react-redux",
+      "jest",
+      "node",
+      "webdriverio/sync",
```

Comment:
the docs mention `webdriverio/async`
is there a difference?

Reply:
Yeah, you are right. `webdriverio/async` should be here.